packages/react/src/_popover/Popover.tsx

```diff
@@ -79,43 +83,31 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 			// Adds distance between the reference and floating element
 			// https://floating-ui.com/docs/offset
 			offset(offsetOption),
-			// Changes the placement of the floating element in order to keep it in view
-			// https://floating-ui.com/docs/flip
-			flip({ padding: DEFAULT_OFFSET }),
 			// Allows you to change the size of the floating element
 			// https://floating-ui.com/docs/size
+			shift({ padding: MIN_SIDE_GUTTER_WIDTH }),
 			size({
 				padding: DEFAULT_OFFSET, // Prevents the floating element hit the edge of the screen
-				apply({
-					availableWidth,
-					availableHeight: _availableHeight,
-					elements,
-					rects,
-				}) {
-					// Popovers can have a predefined max-height if there is enough room on the screen
-					const maxHeight =
```

Comment:
I think removing this is generally better, because the old behaviour that the calendar widget's height would change as you scroll as more space became available is pretty weird. It creating scrollbars was also not ideal.